src/scss/components/_button.scss

```diff
@@ -49,29 +48,19 @@
 		background: helpers.$gradient-yellow-dark;
 		opacity: 0;
 		transition: opacity 0.5s ease-in-out;
-		@include helpers.zIndex(layer_negative);
```

Comment:
I think you should add layer with value 1, and add it to this place.
if you use position, please use z index with it, It will help you to avoid potential bugs.
the same situation like with pointing exact property for transition to animate

Reply:
Done